DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/AutocadDocumentManager.cs

```diff
@@ -0,0 +1,148 @@
+using System.Diagnostics;
+using System.Text;
+using Autodesk.AutoCAD.DatabaseServices;
+
+namespace Speckle.Connectors.Autocad.HostApp;
+
+public class AutocadDocumentManager
+{
+  private const string SPECKLE_KEY = "Speckle_DUI3";
+  private const string SPECKLE_MODEL_CARDS_KEY = "Speckle_DUI3_Model_Cards";
+
+  /// <summary>
+  /// Returns all the speckle model cards present in the current document.
+  /// </summary>
+  /// <param name="doc"></param>
+  /// <returns></returns>
+  public string? ReadModelCards(Document doc)
```

Comment:
Are we in conrol of this? If doc can be null we should be showing this with nullability, if it cannot then the null check makes no sense...

Reply:
Doc is not null here, so I've removed the unecessary check, but the method is returning null on a few conditions and a caught exception, so I'm leaving the nullable string as is.